<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SchemeEditorMainWindow</class>
 <widget class="QMainWindow" name="SchemeEditorMainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>886</width>
    <height>512</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Atlas Scheme Editor</string>
  </property>
  <property name="windowIcon">
   <iconset resource="images.qrc">
    <normaloff>:/images/theme-editor-icon.png</normaloff>:/images/theme-editor-icon.png</iconset>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="0">
     <widget class="GraphicsView" name="graphicsView"/>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>886</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="actionSelectLibrary"/>
    <addaction name="separator"/>
    <addaction name="actionSave"/>
    <addaction name="actionLoad"/>
   </widget>
   <widget class="QMenu" name="menuView">
    <property name="title">
     <string>View</string>
    </property>
    <addaction name="actionZoomIn"/>
    <addaction name="actionZoomOut"/>
    <addaction name="separator"/>
    <addaction name="actionFitToScene"/>
    <addaction name="separator"/>
    <addaction name="actionClearScene"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuView"/>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <property name="movable">
    <bool>false</bool>
   </property>
   <property name="iconSize">
    <size>
     <width>32</width>
     <height>32</height>
    </size>
   </property>
   <property name="toolButtonStyle">
    <enum>Qt::ToolButtonFollowStyle</enum>
   </property>
   <property name="floatable">
    <bool>false</bool>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actionSelectLibrary"/>
   <addaction name="separator"/>
   <addaction name="actionSave"/>
   <addaction name="actionLoad"/>
   <addaction name="separator"/>
   <addaction name="actionZoomIn"/>
   <addaction name="actionZoomOut"/>
   <addaction name="actionFitToScene"/>
   <addaction name="separator"/>
   <addaction name="actionClearScene"/>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <widget class="QToolBar" name="toolBar">
   <property name="windowTitle">
    <string>toolBar</string>
   </property>
   <property name="autoFillBackground">
    <bool>false</bool>
   </property>
   <property name="allowedAreas">
    <set>Qt::LeftToolBarArea|Qt::RightToolBarArea</set>
   </property>
   <property name="iconSize">
    <size>
     <width>50</width>
     <height>50</height>
    </size>
   </property>
   <property name="toolButtonStyle">
    <enum>Qt::ToolButtonTextUnderIcon</enum>
   </property>
   <property name="floatable">
    <bool>false</bool>
   </property>
   <attribute name="toolBarArea">
    <enum>LeftToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <action name="actionSelectLibrary">
   <property name="icon">
    <iconset resource="images.qrc">
     <normaloff>:/images/Cabinet-icon.png</normaloff>:/images/Cabinet-icon.png</iconset>
   </property>
   <property name="text">
    <string>Select library</string>
   </property>
  </action>
  <action name="actionSave">
   <property name="icon">
    <iconset resource="images.qrc">
     <normaloff>:/images/Save-icon.png</normaloff>:/images/Save-icon.png</iconset>
   </property>
   <property name="text">
    <string>Save</string>
   </property>
   <property name="toolTip">
    <string>Save to configuration file</string>
   </property>
  </action>
  <action name="actionLoad">
   <property name="icon">
    <iconset resource="images.qrc">
     <normaloff>:/images/Folder-Documents-icon.png</normaloff>:/images/Folder-Documents-icon.png</iconset>
   </property>
   <property name="text">
    <string>Load</string>
   </property>
   <property name="toolTip">
    <string>Load configuration from file</string>
   </property>
  </action>
  <action name="actionZoomIn">
   <property name="icon">
    <iconset resource="images.qrc">
     <normaloff>:/images/Zoom-In-icon.png</normaloff>:/images/Zoom-In-icon.png</iconset>
   </property>
   <property name="text">
    <string>Zoom in</string>
   </property>
  </action>
  <action name="actionZoomOut">
   <property name="icon">
    <iconset resource="images.qrc">
     <normaloff>:/images/Zoom-Out-icon.png</normaloff>:/images/Zoom-Out-icon.png</iconset>
   </property>
   <property name="text">
    <string>Zoom out</string>
   </property>
  </action>
  <action name="actionClearScene">
   <property name="icon">
    <iconset resource="images.qrc">
     <normaloff>:/images/Delete-icon.png</normaloff>:/images/Delete-icon.png</iconset>
   </property>
   <property name="text">
    <string>Clear scene</string>
   </property>
  </action>
  <action name="actionFitToScene">
   <property name="icon">
    <iconset resource="images.qrc">
     <normaloff>:/images/Search-File-icon.png</normaloff>:/images/Search-File-icon.png</iconset>
   </property>
   <property name="text">
    <string>Fit to scene</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>GraphicsView</class>
   <extends>QGraphicsView</extends>
   <header>graphicsview.h</header>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="images.qrc"/>
 </resources>
 <connections/>
</ui>
